plugin.rb

```diff
@@ -73,25 +78,45 @@ class Engine < ::Rails::Engine
   end
 
   add_to_serializer(:post, :reactions) do
```

Comment:
the post serializer was modified a fair bit, is it covered? a "GET" of posts that looks at the reactions?

Reply:
Yeah there are already tests for the serializer and for requests. I will go and double check though and make sure I've got the coverage needed.